tests/commonlib/io_utils.py

```diff
@@ -129,17 +131,62 @@ def exec_command(container_name: str, command: str, param_value: str, resource:
         if command == 'chmod':
             os.chmod(path=resource, mode=int(param_value, base=8))
         elif command == 'chown':
-            uid_gid = param_value.split(':')
-            if len(uid_gid) != 2:
-                raise Exception(
-                    "User and group parameter shall be separated by ':' ")
-            shutil.chown(path=resource, user=uid_gid[0], group=uid_gid[1])
+            try:
+                uid, gid = param_value.split(':')
+            except ValueError as exc:
+                raise Exception("User and group parameter shall be separated by ':' ") from exc
+
+            FsClient.add_users_to_node([uid, gid], in_place=True)
```

Comment:
@yashtewari, add_users_to_node will be executed always. Can we solve this in more elegant way? For example to ask either user /group exists and if not to call the function.

Reply:
We can, but it would be over optimization. We have much more critical optimizations to do at the moment, compared to saving a few milliseconds here.